modules/reference/pages/data-transform-rust-sdk.adoc

```diff
@@ -0,0 +1,130 @@
+= Rust SDK for Data Transforms
+:description: Work with data transforms using Rust.
+
+Rust SDK reference documentation for Redpanda data transforms.
+
+== Data transforms
```

Comment:
Is this heading required? we could remove this heading and bump the others to avoid too much nesting.